pkg/drivers/kic/kic.go

```diff
@@ -92,6 +92,27 @@ func (d *Driver) Create() error {
 			ContainerPort: constants.DockerDaemonPort,
 		},
 	)
+
+	exists, err := oci.ContainerExists(d.OCIBinary, params.Name)
+	if err != nil {
+		glog.Warningf("failed to check if container already exists: %v", err)
+	}
+	if exists {
+		// if container was created by minikube it is safe to delete and recreate it.
+		if oci.IsCreatedByMinikube(d.OCIBinary, params.Name) {
+			glog.Info("Found already existing abandoned minikube container, will try to delete.")
+			if err := oci.DeleteContainer(d.OCIBinary, params.Name); err != nil {
+				glog.Errorf("Failed to delete a conflicting minikube container %s. You might need to restart your %s daemon and delete it manually and try again.", params.Name, params.OCIBinary)
+				return errors.Wrapf(err, "deleting already in-use mini-created container %s", params.Name)
```

Comment:
Does DeleteContainer fail if the container has been removed already? 

If so, we should just log the error and try CreateContainer anyways, rather than adding a possible race against another process which may have deleted the container in the mean time.

Reply:
Also, "mini-created" is not be meaningful to users. Is there another way to state this?

Reply:
good point